src/models/GoalItem.ts

```diff
@@ -28,10 +28,12 @@ export interface GoalItem {
   link: string | null;
   participants: IParticipant[];
   rootGoalId: string;
-  timeBudget: {
-    perDay: string | null;
-    perWeek: string | null;
-  } | null;
+  timeBudget:
```

Comment:
you can write this instead
timeBudget?:
{
        perDay: string;
        perWeek: string;
      }